src/home/celebration/NftReward.tsx

```diff
@@ -83,9 +83,9 @@ export default function NftRewardBottomSheet() {
           contractAddress: nftCelebration.contractAddress,
           remainingDays: differenceInDays(rewardExpirationDate, Date.now()),
         })
-      }
 
-      dispatch(nftRewardDisplayed())
+        dispatch(nftRewardDisplayed())
```

Comment:
can this be easily tested? looks like patch coverage is failing

Reply:
unfortunately I don't know the way to simulate bottom sheet close.
the only way I see is to trick coverage bot for now.